packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
+  "Email": "E-post",
+  "Files Upload": "Uppladdning av filer",
+  "HomePage.notification.newsLetter.success": "Prenumererade på nyhetsbrevet",
+  "New entry": "Nytt inlägg",
+  "Password": "Lösenord",
+  "Provider": "Leverantör",
+  "ResetPasswordToken": "Återställ lösenordstoken",
+  "Role": "Roll",
+  "Roles & Permissions": "Roller och behörigheter",
+  "Settings Manager": "Inställningshanterare",
```

Comment:
Not sure if you would use the suffix "hanterare" here. If it is just the settings, then plain "Inställningar" would be correct.

Reply:
> Not sure if you would use the suffix "hanterare" here. If it is just the settings, then plain "Inställningar" would b

Updated to that.